src/helpers/helpers.ts

```diff
@@ -60,3 +61,22 @@ export function awsOneYear(): string {
 export function delay(ms: number) {
     return new Promise((resolve) => setTimeout(resolve, ms));
 }
+
+export function isValidUnixTimestamp(value: string) {
+    const timestamp = Number(value);
+    if (isNaN(timestamp)) {
+        return false;
+    }
+
+    // If the timestamp is in seconds (10 digits)
+    if (value.length === 10) {
```

Comment:
maybe we can avoid magic numbers by using some constants:
const SECONDS_TIMESTAMP_LENGTH = 10;
const MILLISECONDS_TIMESTAMP_LENGTH = 13;

Reply:
done